.secrets.baseline

```diff
@@ -135,7 +135,17 @@
         "verified_result": null
       }
     ],
-    "transforms/universal/tokenization/test-data/ds02/expected/metadata.json": [
+    "transforms/universal/tokenization/python/test-data/ds02/expected/metadata.json": [
+      {
+        "hashed_secret": "ec36b3e3e61563a3002cbac1fb1407757b31b8ee",
+        "is_secret": false,
+        "is_verified": false,
+        "line_number": 55,
+        "type": "Base64 High Entropy String",
+        "verified_result": null
+      }
+    ],
+    "transforms/universal/tokenization/ray/test-data/ds02/expected/metadata.json": [
```

Comment:
Why is this here?

Reply:
this is not new, it needed to be added because  test-data/expected/metadata.json was copied from ray to python dirs.  For whatever reason, pre-commit seems to think the following is a secret

`"name": "/Users/xdang/00proj/04-FM/01_code/fm-data-engineering/transforms/universal/tokenization/output/ds02",`